tools/wavetool.py

```diff
@@ -81,6 +81,22 @@ def generate_wov():
     data[:][data.shape[0] - sine_data2.shape[0]:data.shape[0]] = sine_data2
     return data
 
+def generate_chirp():
+    """
+    Generate chirp wave.
+    ``y(t) = A * sin(2 * pi * f(t) * t + phi)``, where
+    ``f(t) = f_start + chirp_rate * time``,
+    ``chirp_rate = (f_end - f_start) / duration``
+
+    Returns
+    ----------
+    Real n-D Array with shape ``(duration * sample_rate, channel)``
+    """
+    time = np.arange(0, cmd.duration[0], 1.0 / cmd.sample_rate)
+    freq = cmd.freq[0] + np.arange(len(time)) * (cmd.freq[1] - cmd.freq[0]) / len(time)
+    data = cmd.amp[0] * np.sin(2 * np.pi * freq * time + cmd.phase[0])
+    return np.reshape(np.repeat(data, cmd.channel),[len(data), cmd.channel])
```

Comment:
generic comment on this file: you should look into generating different waveforms for each channel. Having the same chirp on all channels isn't always useful. Likewise having two frequencies for left and right sinewaves, or different amplitudes is VERY useful.